src/configurations/destinations/comscore/db-config.json

```
@@ -12,41 +12,59 @@
       "blacklistedEvents",
       "whitelistedEvents",
       "eventFilteringOption",
+      "fieldMapping",
       "oneTrustCookieCategories",
       "consentManagement",
       "ketchConsentPurposes"
     ],
     "excludeKeys": [],
-    "supportedSourceTypes": ["android", "ios"],
+    "supportedSourceTypes": ["android", "ios", "web"],
     "supportedMessageTypes": {
       "device": {
         "android": ["identify", "track", "screen"],
-        "ios": ["identify", "track", "screen"]
+        "ios": ["identify", "track", "screen"],
+        "web": ["page"]
       }
     },
     "supportedConnectionModes": {
       "android": ["device"],
-      "ios": ["device"]
+      "ios": ["device"],
+      "web": ["cloud"]
```

```suggestion
      "web": ["device"]
```
shouldn't this be device ?